.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Per Mattermost guidelines, GitHub issues are for bug reports: <http://www.mattermost.org/filing-issues/>.
```

Comment:
What will happen with the existing template https://github.com/mattermost/mattermost-mobile/blob/master/ISSUE_TEMPLATE.md? 

I think it would be good to include the Environment Information here
Device Name:
OS Version:
Mattermost App Version:
Mattermost Server Version:

Reply:
PTAL
thanks for your review